pkg/controller/drain/drain_controller.go

```diff
@@ -319,7 +326,15 @@ func (ctrl *Controller) syncNode(key string) error {
 		ctrl.logNode(node, "initiating drain")
 		if err := drain.RunNodeDrain(drainer, node.Name); err != nil {
 			ctrl.logNode(node, "Drain failed. Waiting 1 minute then retrying. Error message from drain: %v", err)
```

Comment:
w are now waiting for either drainRequeueDelay or drainRequeueFailingDelay amount of time. Lets move this log inside `if` so that correct waiting time gets logged.

Reply:
Sorry, for some reason I thought I pushed the fix a few days ago, but I guess I never actually ran the push command. Should be fixed now